spec/classes/check_mk_agent_config_spec.rb

```diff
@@ -65,9 +65,10 @@
         it {
           is_expected.to contain_file('/etc/check_mk/encryption.cfg').with(
             'ensure'  => 'file',
-            'mode'    => '0600',
-            'content' => %r{PASSPHRASE=SECRET\n}
+            'mode'    => '0600'
           )
+          content = catalogue.resource('file', '/etc/check_mk/encryption.cfg').parameters[:content]
+          expect(content).to include("PASSPHRASE=SECRET\n")
```

Comment:
I think you can check it with `'content' => sensitive('PASSPHRASE=SECRET\n')` but I don't know if that supports partial matches.